src/modules/assessments/components/AssessmentTitle.tsx

```diff
@@ -81,7 +89,17 @@ const AssessmentTitle = ({
           {assessmentTitle}
           {': '}
         </b>
-        {projectName}
+        <RouterLink
+          to={generateSafePath(Routes.ENROLLMENT_DASHBOARD, {
+            enrollmentId,
+            clientId,
+          })}
+          openInNew
+          sx={{ display: 'inline-flex' }}
+        >
+          {projectName} ({parseAndFormatDate(entryDate)} -{' '}
+          {exitDate ? parseAndFormatDate(exitDate) : 'Active'})
```

Comment:
was there a reason not to use `parseAndFormatDateRange(entryDate, exitDate)` here?

Reply:
No, no reason 🤦‍♀️